components/lib/crash/src/main/java/mozilla/components/lib/crash/service/MozillaSocorroService.kt

```diff
@@ -174,7 +174,7 @@ class MozillaSocorroService(
                 return id
             }
         } catch (e: IOException) {
-            logger.error("failed to send report to Socorro", e)
+            logger.error("failed to send report to Socorro with " + conn?.responseCode, e)
```

Comment:
Watch out, it looks like asking for the response code can throw an `IOException` too: https://cs.android.com/android/platform/superproject/+/master:external/okhttp/okhttp-urlconnection/src/main/java/com/squareup/okhttp/internal/huc/HttpURLConnectionImpl.java;l=537-539?q=HttpUrlCOnnection

Reply:
oh, I did not see that.  I'll update.

Reply:
Thanks for catching this.